setup.py

```diff
@@ -45,6 +45,7 @@ def get_long_description():
     license="AGPLv3",
     url="https://github.com/man-group/notebooker",
     packages=find_packages(exclude=["tests", "tests.*", "benchmarks"]),
+    package_data={'': ["CHANGELOG.md", "**/*.tpl"]},
```

Comment:
Add these to `MANIFEST.in` instead, unless the `{'': ...}` syntax does something magic?

Reply:
Good point - fixed